src/javascript/binary/static/common/common.utility.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Created by qingwei on 10/13/15.
+ */
+
+var CommonUtility = (function(){
```

Comment:
Can I changed it to StringHelper?
May I know why is it a bad variable name? So I can avoid in the future